workers/unity/Assets/Gdk/Core/Attributes/RemoveAtEndOfTick.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using Unity.Entities;
+
+namespace Improbable.Gdk.Core
+{
+    /// <summary>
+    /// Any component with this attribute will be removed from all entities by the CleanReactiveComponentSystem
+    /// Can be added to components extending either <see cref="IComponentData" /> or <see cref="ISharedComponentData" />
+    /// </summary>
+    [AttributeUsage(AttributeTargets.Struct | AttributeTargets.Class, AllowMultiple = false, Inherited = false)]
```

Comment:
Just wondering, what kind of classes will this be on?

Reply:
I think that is a mistake. Will remove it. (Could conceivably add it to components but that will break if done right now)